starter-code/src/clue.js

```diff
@@ -85,10 +85,77 @@ name: Patio
 */
 
 // Characters Collection
-var charactersArray = [];
+//var charactersArray = [];
 
 // Rooms' Collection
-var roomsArray = [];
+//var roomsArray = [];
 
 // Weapons Collection
-var weaponsArray = [];
+//var weaponsArray = [];
+
+
+
+
+// Characters Collection
+var charactersArray = [
+  
+    {first_name: "Jacob", last_name: "Green", color: "Green", description: "Has a lot of connections", age: "45", image: "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg", occupation: "Entrepreneur"},
+  
+  {first_name: "Doctor", last_name: "Orchid", color: "White", description: "PhD in plant toxicology. Adopted daughter of Mr. Boddy", age: "26", image: "http://www.radiotimes.com/uploads/images/Original/111967.jpg", occupation: "scientist"},
+  
+  {first_name: "Victor", last_name: "Plum", color: "Purple", description: "Billionare video game designer", age: "22", image: "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg", occupation: "Designer"},
+  
+  {first_name: "Kasandra", last_name: "Scarlet", color: "Red", description: "She is an A-list movie star with a dark past", age: "31", image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg", occupation: "Actor"},
+  
+  {first_name: "Eleanor", last_name: "Peacock", color: "Blue", description: "She is from a wealthy family and uses her status and money to earn popularity", age: "36", image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg", occupation: "Socialite"},
+  
+  {first_name: "Jack", last_name: "Mustard", color: "Yellow", description: "He is a former football player who tries to get by on his former glory", age: "62",image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg", occupation: "Retired Football player"}];
+  
+  // Rooms' Collection
+  
+  var roomsArray = [ 
+  {name: "Dinning Room"},
+  {name: "Conservatory"},
+  {name: "Kitchen"},
+  {name: "Study"},
+  {name: "Library"},
+  {name: "Billiard Room"},
+  {name: "Lounge"},
+  {name: "Ballroom"},
+  {name: "Hall"},
+  {name: "Spa"},
+  {name: "Living Room"},
+  {name: "Observatory"},
+  {name: "Theater"},
+  {name: "Guest House"},
+  {name: "Patio"},
+  ];
+  
+  // Weapons Collection
```

Comment:
This part is also incorrect, again you decided to hard code the array... It  should have been:
```js
// Weapons
var rope        = { name: 'rope',         weight: 10 };
var knife       = { name: 'knife',        weight: 8 };
var candlestick = { name: 'candlestick',  weight: 2 };
var dumbbell    = { name: 'dumbbell',     weight: 30 };
var poison      = { name: 'poison',       weight: 2 };
var axe         = { name: 'axe',          weight: 15 };
var bat         = { name: 'bat',          weight: 13 };
var trophy      = { name: 'trophy',       weight: 25 };
var pistol      = { name: 'pistol',       weight: 20 };
```